CMP1903M Assessment 1 Base Code/Program.cs

```diff
@@ -13,24 +13,78 @@ static void Main()
         {
             //Local list of integers to hold the first five measurements of the text
             List<int> parameters = new List<int>();
-
             //Create 'Input' object
+            Input textobj = new();
             //Get either manually entered text, or text from a file
+            bool choice_made;
+            choice_made = false;
+            while (choice_made == false)
+            {
+                Console.WriteLine("Please choose a method of text input.  For Manual text enter [ M ]   or   For File text enter [ F ] >>>: ");
+                string choice = Console.ReadLine();
 
-
+                if (choice == "M" | choice == "m")
```

Comment:
```suggestion
                if (choice == "M" || choice == "m")
```
Be careful with this, a single `|` is a bitwise operation, not a boolean OR